libs/datasets/sources/fips_population.csv

```diff
@@ -3141,81 +3141,81 @@ WY,Teton County,56039,23464.0
 WY,Uinta County,56041,20226.0
 WY,Washakie County,56043,7805.0
 WY,Weston County,56045,6927.0
-PR,Adjuntas Municipio,72001,
-PR,Aguada Municipio,72003,
-PR,Aguadilla Municipio,72005,
-PR,Aguas Buenas Municipio,72007,
-PR,Aibonito Municipio,72009,
-PR,Añasco Municipio,72011,
-PR,Arecibo Municipio,72013,
-PR,Arroyo Municipio,72015,
-PR,Barceloneta Municipio,72017,
-PR,Barranquitas Municipio,72019,
-PR,Bayamón Municipio,72021,
-PR,Cabo Rojo Municipio,72023,
-PR,Caguas Municipio,72025,
-PR,Camuy Municipio,72027,
-PR,Canóvanas Municipio,72029,
-PR,Carolina Municipio,72031,
-PR,Cataño Municipio,72033,
-PR,Cayey Municipio,72035,
-PR,Ceiba Municipio,72037,
-PR,Ciales Municipio,72039,
-PR,Cidra Municipio,72041,
-PR,Coamo Municipio,72043,
-PR,Comerío Municipio,72045,
-PR,Corozal Municipio,72047,
-PR,Culebra Municipio,72049,
-PR,Dorado Municipio,72051,
-PR,Fajardo Municipio,72053,
-PR,Florida Municipio,72054,
-PR,Guánica Municipio,72055,
-PR,Guayama Municipio,72057,
-PR,Guayanilla Municipio,72059,
-PR,Guaynabo Municipio,72061,
-PR,Gurabo Municipio,72063,
-PR,Hatillo Municipio,72065,
-PR,Hormigueros Municipio,72067,
-PR,Humacao Municipio,72069,
-PR,Isabela Municipio,72071,
-PR,Jayuya Municipio,72073,
-PR,Juana Díaz Municipio,72075,
-PR,Juncos Municipio,72077,
-PR,Lajas Municipio,72079,
-PR,Lares Municipio,72081,
-PR,Las Marías Municipio,72083,
-PR,Las Piedras Municipio,72085,
-PR,Loíza Municipio,72087,
-PR,Luquillo Municipio,72089,
-PR,Manatí Municipio,72091,
-PR,Maricao Municipio,72093,
-PR,Maunabo Municipio,72095,
-PR,Mayagüez Municipio,72097,
-PR,Moca Municipio,72099,
-PR,Morovis Municipio,72101,
-PR,Naguabo Municipio,72103,
-PR,Naranjito Municipio,72105,
-PR,Orocovis Municipio,72107,
-PR,Patillas Municipio,72109,
-PR,Peñuelas Municipio,72111,
-PR,Ponce Municipio,72113,
-PR,Quebradillas Municipio,72115,
-PR,Rincón Municipio,72117,
-PR,Río Grande Municipio,72119,
-PR,Sabana Grande Municipio,72121,
-PR,Salinas Municipio,72123,
-PR,San Germán Municipio,72125,
-PR,San Juan Municipio,72127,
-PR,San Lorenzo Municipio,72129,
-PR,San Sebastián Municipio,72131,
-PR,Santa Isabel Municipio,72133,
-PR,Toa Alta Municipio,72135,
-PR,Toa Baja Municipio,72137,
-PR,Trujillo Alto Municipio,72139,
-PR,Utuado Municipio,72141,
-PR,Vega Alta Municipio,72143,
-PR,Vega Baja Municipio,72145,
-PR,Vieques Municipio,72147,
-PR,Villalba Municipio,72149,
-PR,Yabucoa Municipio,72151,
-PR,Yauco Municipio,72153,
+PR,Adjuntas Municipio,72001,17363.0
```

Comment:
Can you add this to https://github.com/covid-projections/covid-data-public/blob/master/data/misc/fips_population.csv as well?

Reply:
We should probably remove this file and just rely on covid-data-public as well, but no need to do that in this PR

Reply:
Ok, will add there!